tools/SetupFlow/DevHome.SetupFlow/Models/InstallPackageTask.cs

```diff
@@ -282,11 +282,11 @@ private void ReportAppSelectedForInstallEvent()
 
     private void ReportAppInstallSucceededEvent()
     {
-        TelemetryFactory.Get<ITelemetry>().Log("AppInstall_InstallSucceeded", LogLevel.Critical, new AppInstallEvent(_package.Id, _package.CatalogId), _activityId);
+        TelemetryFactory.Get<ITelemetry>().Log("AppInstall_InstallSucceeded", LogLevel.Critical, new AppInstallResultEvent(_package.Id, _package.CatalogId), _activityId);

```

Comment:
Please modify the name to use two `_'s` to keep in line with other event names.  Something like `AppPage_AppInstall_InstallSucceeded`

Reply:
The change in this PR address the telemetry PartA property. Changing the name may require further consideration. Check https://github.com/microsoft/devhome/pull/2809#discussion_r1591384308 for more details.